docs/user/installing/configuring-auth/auth-pam.md

```diff
@@ -103,6 +111,14 @@ auth:
 | `allowPublicClientWithoutPKCE` | Allow public clients to skip PKCE requirement. **Security Warning**: Only enable for testing | `false` |
 | `accessTokenExpiration` | Expiration duration for access tokens and ID tokens (e.g., `1h`, `30m`) | `1h` |
 | `refreshTokenExpiration` | Expiration duration for refresh tokens (e.g., `168h`, `720h`) | `168h` |
+| `branding` | Branding configuration for the login form | See defaults below |
+| `branding.issuerName` | Display name shown on the login form | `"Flight Control"` |
+| `branding.logoLight` | Logo image for light theme (URL or base64 data URI). If empty, uses default embedded logo | Empty (uses default) |
+| `branding.logoDark` | Logo image for dark theme (URL or base64 data URI). If empty, uses default embedded logo | Empty (uses default) |
+| `branding.lightPrimaryColor` | Primary color for light theme (hex color code) | `"#007bff"` |
+| `branding.lightSecondaryColor` | Secondary color for light theme (hex color code) | `"#666"` |
+| `branding.darkPrimaryColor` | Primary color for dark theme (hex color code) | `"#4a9eff"` |
+| `branding.darkSecondaryColor` | Secondary color for dark theme (hex color code) | `"#b0b0b0"` |
```

Comment:
appreciate the PR! do we want all these knobs here I get we need more control but I wonder if @celdrake or @rawagner has ideas. when I first tagged this templating I didn't consider the loss of control/review by UI/UX team. I would really like to wait for additional feedback.

Reply:
In fc ui you can switch themes (at runtime) , to be consistent I did the same , also keep in mind that we are serving a single html , if we want something more complex we would need a dedicated nginx serving just pam issuer's UI

Reply:
True, you can switch between up- and downstream themes at runtime (I suspect to enable testing / demo theming). But the two themes are fully defined and embedded. Why make this user-configurable and thereby create API baggage without need?